recipe/meta.yaml

```diff
@@ -46,25 +46,25 @@ requirements:
    - ptyprocess >=0.5  # [win]
```

Comment:
I wonder if it's still relevant. See https://github.com/spyder-ide/spyder/pull/16766#issuecomment-963601853 and https://github.com/conda-forge/spyder-feedstock/pull/119

Reply:
maybe not, but I'll leave the research effort for the next update.